packages/shylu/shylu_dd/bddc/src/shylu_PartitionOfUnityBDDC.h

```diff
@@ -85,37 +88,39 @@ template <class SX,
   }
 
   PartitionOfUnity(LO numNodes,
-                   const GO* nodeGlobalIDs,
-                   const std::vector< std::vector<LO> > & subNodeBegin,
-                   const std::vector< std::vector<LO> > & subNodes,
-                   LO** subRowBegin,
-                   LO** subColumns,
-                   SX** subValues,
-                   LO spatialDim,
-                   RCP<Teuchos::ParameterList> & Parameters,
-                   RCP<const Teuchos::Comm<int> > Comm) :
-  m_numNodes(numNodes),
+		   const GO* nodeGlobalIDs,
+		   const std::vector< std::vector<LO> > & subNodeBegin,
+		   const std::vector< std::vector<LO> > & subNodes,
+		   LO spatialDim,
+		   RCP<Teuchos::ParameterList> & Parameters,
+		   RCP<const Teuchos::Comm<int> > Comm,
+		   RCP<Tpetra::Distributor> & distributor,
```

Comment:
Is this an output or in/out argument?  If not, please pass the RCP by value, not by reference (i.e., remove the `&`).

Reply:
Done.